src/sidebars/history.ts

```diff
@@ -42,6 +46,26 @@ export class HistoryProvider
     }
   }
 
+  getParent(
+    element: TestStep | TestRecord,
+  ): vscode.ProviderResult<TestStep | TestRecord> {
+    if (isTestRecord(element)) {
+      return null;
+    } else {
```

Comment:
The `else` block is superfluous, since you'll exit in a truthy `if`. Can remove this wrapper 🧹 and the main code block will save itself one indentation 🎉 .